soap/src/java/com/zimbra/soap/type/ShareInfo.java

```diff
@@ -128,72 +133,85 @@ public ShareInfo() {
 
     public void setOwnerId(String ownerId) { this.ownerId = ownerId; }
 
+    @GraphQLQuery(name=GqlConstants.OWNER_ID, description="owner id")
```

Comment:
Add `@GraphQLNonNull` above each query annotation for all of the following except mountpoint id.

Reply:
added